packages/gatsby-theme-carbon/src/components/Code/Code.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import Highlight, { defaultProps } from 'prism-react-renderer';
+import { Highlight, themes } from "prism-react-renderer"
```

Comment:
Single quotes here.

Reply:
Looks like you are importing `themes` here but not using it anywhere?

Reply:
Brought back the defaultProps in this component.